src_kt/java/arcs/storage/DirectStore.kt

```diff
@@ -136,7 +136,7 @@ class DirectStore internal constructor(
         version: Int,
         channel: Int?
     ) {
-        if (modelChange.isEmpty() && otherChange?.isEmpty() != false) return
+        if (modelChange.isEmpty() && otherChange?.isEmpty() == true) return
```

Comment:
probably don't need `== true` here surely?

Reply:
Since `otherChange` is nullable, and i'm using `?.isEmpty()`, the return type of that is a nullable boolean... which you can't use directly in a boolean expression. :-/